### PR TITLE
centered reset email btn

### PR DESCRIPTION
**Pull request by Humza23**
Fixed the reset via email button text not being centered. The text is now centered.

Fixes # (issue)

Reset via email button text is now centered.

Type of change
Styling

Checklist:
[ x] My code follows the style guidelines of this project
[ x] I have performed a self-review of my own code
[x ] I have removed unnecessary comments/console logs from my code
[x ] I have made corresponding changes to the documentation if necessary (optional)
[x ] My changes generate no new warnings
[x ] I have checked my code and corrected any misspellings
[ x] No duplicate code left within changed files
[ x] Size of pull request kept to a minimum
[x ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 1]
```diff
@@ -157,6 +157,9 @@
         background: @button;
         height: 2.6rem;
         width: 100%;
+        display: flex;
```
Comment (CrashOverride1987) [inline src/components/pages/Login/_LoginContainerStyled.less:160]: Why?

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 2]
```diff
@@ -157,6 +157,9 @@
...
+        align-items: center;
+        justify-content: center;
```
Comment (CrashOverride1987) [inline src/components/pages/Login/_LoginContainerStyled.less:162]: why?

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 3]
```diff
@@ -157,6 +157,9 @@
         background: @button;
         height: 2.6rem;
         width: 100%;
+        display: flex;
+        align-items: center;
```
Comment (CrashOverride1987) [inline src/components/pages/Login/_LoginContainerStyled.less:161]: This works, but so does align-text

**Review by CrashOverride1987 — APPROVED** [review]

**Review by dancingheart714 — APPROVED** [review]
Nice work Humza - a small, but meaningful update!

**Pull request merged**